lineage/states/StateDistributionGamma.py

```diff
@@ -73,8 +76,10 @@ def estimator(self, x, gammas):
         gamma_obs_censor = x[:, 2]
 
         b_mask = np.isfinite(bern_obs)
-        g_mask = np.isfinite(γ_obs)
-
+        # Both unoberved and dead cells should be removed from gamma
+        g_mask = np.logical_and(np.isfinite(γ_obs), bern_obs == 1)
+        # at least have 10 cells after masking
+        assert np.count_nonzero(g_mask) >= 3
         # Handle an empty state
         if np.sum(gammas[b_mask]) == 0.0:
```

Comment:
Is this assertion reasonable? I mean, do we expect the tests to always pass if there are at least 3 cells for estimation? I made it 10 first but 9 tests failed! @aarmey

Reply:
Yes. One of the failures seems to be from all the cells being dead and therefore eliminated. Set the emissions in test_BW so that there's essentially never cell death.